jetstream/cli.py

```diff
@@ -129,6 +171,18 @@ def run(project_id, dataset_id, date, experiment_slug, config_file):
     thrown during some experiment analyses. This ensures that the Airflow task will
     not retry the task and run all of the analyses again.
     """
+    if argo:
+        return submit_workflow(
+            project_id,
+            zone,
+            cluster_id,
+            RUN_WORKFLOW,
+            {"date": date.strftime("%Y-%m-%d")},
+            monitor_status=monitor_status,
+        )
+
+    # run locally
```

Comment:
I think we still want to be able to simply `run`, `rerun` and do all the CLI stuff locally.